platform-cloud/docs/studios/add-studio.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Add a Studio"
```

Comment:
```suggestion
title: "Create a Studio"
```
I found this page really hard to follow. Looking at the whole PR, I think it's due to the explanation and guide material ending up on the same page. 

I've made numerous small edits, some are editorial, while others are more about reshaping the page into a guide supplemented with explanations.